packages/roosterjs-content-model/lib/editor/plugins/PastePlugin/WordDesktop/handleWordDesktopPaste.ts

```diff
@@ -22,6 +22,11 @@ export function handleWordDesktop(ev: ContentModelBeforePasteEvent) {
     setProcessor(ev.domToModelOption, 'element', wordDesktopElementProcessor);
     addParser(ev.domToModelOption, 'block', removeNonValidLineHeight);
     addParser(ev.domToModelOption, 'listLevel', listLevelParser);
+    addParser(ev.domToModelOption, 'listItemElement', (format, element) => {
```

Comment:
The callback does not use any other thing than the parameter, so it does not need to be a inline callback. Let's make it a regular function